fix(html): add description list section (#31700)
mdn/content: files/en-us/learn/html/introduction_to_html/html_text_fundamentals/index.md

{{ EmbedLiveSample('Active_learning_Marking_up_an_ordered_list', 700, 500, "", "") }}

### Description

> Description list encloses list of groups of terms and descriptions. Common uses for this element are to implement a glossary or to display metadata (a list of key-value pairs). Let's consider some fishes with their interesting characteristics:

Reviewer: Grammar suggestion:

```suggestion
Description lists enclose groups of terms and descriptions. Common uses for this element are implementing a glossary or displaying metadata (a list of key-value pairs). Let's consider some fishes with their interesting characteristics:
```

```plain
Albacore Tuna
The albacore is a very fast swimmer.

Asian Carp
Asian carp can consume 40% of their body weight in food a day!

Barracuda
Can grow to nearly 2 meters long!
```

> Whole list is wrapped in `<dl>` element. Terms are enclosed in `<dt>` elements and descriptions are enclosed in `<dd>` elements:

Reviewer: ```suggestion
The list is enclosed in a `<dl>` element, terms are enclosed in `<dt>` elements, and descriptions are enclosed in `<dd>` elements:
```
